tests/example.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
+XSS Quick Test
+--STDIN--
+<SCRIPT>alert('XSS')</SCRIPT>
+--FILE--
+<?php
+require 'vendor/autoload.php';
+echo htmLawed(stream_get_contents(STDIN), array('safe'=>1));
+?>
+--EXPECT--
+alert('XSS')
```

Comment:
At one point PHPUnit started using namespaces, which made it incompatible with older versions of PHP. Meanwhile this test format still works.